src/helpers/utils.ts

```diff
@@ -78,13 +78,47 @@ export function getBundler(packageJson: any) {
   return undefined;
 }
 
+function detectNpm(root: string) {
+  return new Promise((res) => {
+    access(resolve(root, 'package-lock.json'), constants.F_OK, (noPackageLock) => {
+      res(!noPackageLock);
+    });
+  });
+}
+
+function detectYarn(root: string) {
+  return !!existsSync(`${root}/yarn.lock`);
+}
+
+function detectPnpm(root: string) {
+  return new Promise((res) => {
+    access(resolve(root, 'pnpm-lock.yaml'), constants.F_OK, (noPnpmLock) => {
+      res(!noPnpmLock);
+    });
+  });
+}
+
+async function installDependencies(root: string) {
+  const [hasNpm, hasYarn, hasPnpm] = await Promise.all([detectNpm(root), detectYarn(root), detectPnpm(root)]);
+  if (!hasNpm && !hasYarn && !hasPnpm) {
+    execCommand('npx lerna bootstrap');
```

Comment:
Well, if we have nothing are we sure that we have lerna?

In case of nothing we should run `npm install` as this has to be available anyway.

Lerna needs to be detected via a `lerna.json`.